.idea/.name

```diff
@@ -0,0 +1 @@
+tbbtalentv2
```

Comment:
It is useful to standardize on our configuration - ref your comment about imports later.

We haven't got it properly working yet - but I think it will be useful for us to share certain .idea config files.